src/features/generic/time-utils.ts

```diff
@@ -118,14 +121,19 @@ namespace TimeUtilities {
    *
    * @returns two previous workdays
    */
-  export const getPreviousTwoWorkdays = (): PreviousWorkdayDates => {
-    const today = DateTime.now();
-    const dayOfWeek = new Date().getDay();
+  export const getPreviousTwoWorkdays = (testValueToday: DateTime = null, testValueDayOfWeek: number = null): PreviousWorkdayDates => {
```

Comment:
Add testValueToday and testValueDayOfWeek to function docs.

Reply:
Also instead of settings these test variables to null use ? syntax in the function parameters. This marks the parameters as optional and makes the code a bit more readable.
```suggestion
  export const getPreviousTwoWorkdays = (testValueToday?: DateTime, testValueDayOfWeek?: number): PreviousWorkdayDates => {
```

Reply:
Done